packages/core/src/services/projects/import.ts

```diff
@@ -0,0 +1,76 @@
+import { env } from '@latitude-data/env'
+import { eq } from 'drizzle-orm'
+
+import { Project, User, Workspace } from '../../browser'
+import { database } from '../../client'
+import { Result, Transaction } from '../../lib'
+import { DocumentVersionsRepository } from '../../repositories'
+import { projects } from '../../schema'
+import { mergeCommit } from '../commits'
+import { createCommit } from '../commits/create'
+import { createNewDocument } from '../documents'
+import { createProject } from './create'
+
+export async function importDefaultProject(
+  {
+    workspace,
+    user,
+  }: {
+    workspace: Workspace
+    user: User
+  },
+  db = database,
+) {
+  if (!('DEFAULT_PROJECT_ID' in env) || env.DEFAULT_PROJECT_ID === undefined) {
+    return Result.ok(null)
+  }
+
+  const defaultProjectId = env.DEFAULT_PROJECT_ID
+
+  const defaultProject = await db.query.projects.findFirst({
+    where: eq(projects.id, defaultProjectId),
+  })
+
+  const defaultProjectDocumentsScope = new DocumentVersionsRepository(
+    defaultProject!.workspaceId,
+  )
+  const defaultDocuments =
+    await defaultProjectDocumentsScope.getDocumentsFromMergedCommits({
+      projectId: defaultProject!.id,
+    })
+  if (defaultDocuments.error) return defaultDocuments
+
+  return Transaction.call<Project>(async (tx) => {
+    const project = await createProject(
+      {
+        workspace,
+        user,
+        name: defaultProject!.name,
+      },
```

Comment:
if this project needs to be created with a merged commit you need to pass a mergedAt option here

Reply:
I'm using the `createNewDocument` service, which requires an unmerged commit. I'm merging it afterwards